libqtile/widget/base.py

```diff
@@ -116,7 +116,8 @@ def open_calendar():
     offsety: int = 0
     defaults = [
         ("background", None, "Widget background color"),
-        ("mouse_callbacks", {}, "Dict of mouse button press callback functions."),
+        ("mouse_callbacks", {},
+         "Dict of mouse button press callback functions. Set to `None` to disable"),
```

Comment:
We probably want to modify `add_callbacks` to reflect that for the current widgets? i.e. not adding callbacks if mouse_callbacks is None.